deployments/charts/kuma/values.yaml

```diff
@@ -194,6 +194,17 @@ kumactl:
     # -- The kumactl image repository
     repository: kumactl
 
+kubectl:
+  # bitnami maintains an image for all k8s versions */ } }
+  # see: https://hub.docker.com/r/bitnami/kubectl */ } }
+  image:
+    # -- The kubectl image registry
+    registry: bitnami
+    # -- The kubectl image repository
+    repository: kubectl
+    # -- The kubectl image tag
+    # tag: latest
```

Comment:
Shouldn't we specify the version of an image here?

Reply:
This is put for reference here, we're actually reading the version from the Cluster Capabilities.